apps/ios/Shared/Model/ChatModel.swift

```diff
@@ -82,19 +86,19 @@ class ItemsModel: ObservableObject {
                 await MainActor.run { showLoadingProgress = true }
             } catch {}
         }
+        let type = ChatType(rawValue: String(chatId.prefix(1)))!
+        let id = Int64(chatId.suffix(from: chatId.index(chatId.startIndex, offsetBy: 1)))!
```

Comment:
this is hacky - you should never need to parse IDs - change loadChat and downstream API types instead